OCast/src/main/java/org/ocast/core/WebSocketProvider.kt

```diff
@@ -61,10 +69,14 @@ open class WebSocketProvider(private val webSocketURL: String, private val sslCo
                 val client = builder.build()
                 val request = Request.Builder().url(webSocketURL).build()
                 webSocket = client.newWebSocket(request, this)
+                true
             } catch (e: Exception) {
                 OCastLog.error("Socket: Error create socket", e)
                 listener.onDisconnected(this, e)
+                false
             }
+        } else {
+            false
```

Comment:
In this case state is `CONNECTED` or `CONNECTING`, should we return `true` instead ? Returning `false` might trigger errors although everything is fine.